src/index.js

```diff
@@ -48,11 +40,10 @@ export default function(deckName) {
 
   models[top_model_id + ''] = model;
 
-  update('update col set models=:models where id=1', { ':models': JSON.stringify(models) });
+  exporter.update('update col set models=:models where id=1', { ':models': JSON.stringify(models) });
 
-  return {
-    addMedia,
-    addCard,
-    save
-  };
+  return ['addCard', 'addMedia', 'save'].reduce((prev, i) => {
```

Comment:
As I showed in https://github.com/ewnd9/anki-apkg-export/pull/7#issuecomment-263780460 it's better to just return an instance of `Exporter` class